docker-compose.yml

```diff
@@ -25,10 +28,12 @@ services:
       - OIDC_SUBJECT_IDENTIFIERS_ENABLED=public,pairwise
       - OIDC_SUBJECT_IDENTIFIERS_PAIRWISE_SALT=youReallyNeedToChangeThis
 
+  #  editor-renderer:
```

Comment:
Deactivated `editor-renderer` and `legacy-editor-renderer` local builds for now since they aren't touched in nearly all cases anyways and will be replaced soon by the frontend. And if you want to change them, you want to start them locally anyways (via `yarn start`). Still: previous code is still here if you need it.